Assets/_GameJamCat/Scripts/Managers/UIManager.cs

```diff
@@ -101,6 +101,7 @@ private void HandleOnStateChange(State state)
                     OnPregameSet();
                     break;
                 case State.Play:
+                    SetCrossHairState(true);
```

Comment:
Remove this, since it's not always true, it also depends on if the Doisser is opened or not.